core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -68,14 +68,13 @@ object ConfigCommand extends Config {
   val BrokerDefaultEntityName = ""
   val BrokerLoggerConfigType = "broker-loggers"
   val BrokerSupportedConfigTypes = ConfigType.all :+ BrokerLoggerConfigType
-  val ZkSupportedConfigTypes = ConfigType.all
+  val ZkSupportedConfigTypes = Seq(ConfigType.User, ConfigType.Broker)
   val DefaultScramIterations = 4096
   // Dynamic broker configs can only be updated using the new AdminClient once brokers have started
   // so that configs may be fully validated. Prior to starting brokers, updates may be performed using
   // ZooKeeper for bootstrapping. This allows all password configs to be stored encrypted in ZK,
   // avoiding clear passwords in server.properties. For consistency with older versions, quota-related
-  // broker configs can still be updated using ZooKeeper at any time. ConfigCommand will be migrated
-  // to the new AdminClient later for these configs (KIP-248).
```

Comment:
I've checked and confirmed that KIP-248 is rejected now. It won't be implemented. So, remove it.

Reply:
Hmm, the comment says that quota related broker configs can still be updated using ZooKeeper at any time. Why is that? I think we should remove this.

Reply:
> Why is that?

For consistency with older versions

> I think we should remove this.

Agree with you. It's a good timing now.